### ISSUE TITLE
Keyboard shortcuts do not actually change focus

### ISSUE THREAD

**Issue by nolanlawson** [nolanlawson/pinafore#1018]
>J and K, when moving between toots, does not yet actually focus the new toot, only styles it differently from the old one? If I turn off virtual cursor with NVDA, J and K don't speak or move DOM focus.

I took a look at the Mastodon implementation, and it correctly changes the focus as you press J and K.

**Comment by nolanlawson** [comment]
So one issue here is that the "active" style just applies a light background to the toot, whereas the default focus style is a focus ring around the toot. I think the light background is more visually appealing, but the focus ring is more consistent with the rest of the UI, so I'm not sure which one to keep.

Probably we should just ditch the light background as it's already visually confusing with DMs anyway, and in the context of DMs themselves, it just overrides the background color instead of having a third color based on DM + active.

**Comment by nolanlawson** [comment]
It looks like the entire hotkey system was built based on this pseudo-active state rather than the real active state. If a toot is truly focused (not "active"), I cannot press F to favorite or X to open a CW.

**Comment by nolanlawson** [comment]
Looking over the original PR (https://github.com/nolanlawson/pinafore/pull/870), yes it just creates its own pseudo-active system and manages the whole thing manually. I will probably have to replace all of this and just delegate to the browser, using `focus()` and `document.activeElement`.
